js/lib/cachetasks.js

```diff
@@ -95,7 +95,17 @@
       case "store":
         try {
           response = yield fetch(request);
-          yield cache.put(request, response.clone());
+
+          // Create response with timestamp appended.
+          if (cacheName == "directory_links") {
```

Comment:
I don't thinks we don't need to do this, as the HTTP response will contain a time stamp:

<img width="690" alt="screenshot 2015-10-16 22 17 53" src="https://cloud.githubusercontent.com/assets/870154/10556550/e2785514-7453-11e5-80c2-4c0eb118abf1.png">

If we need a better timestamp, we should generate it on the server.